app/assets/stylesheets/components/_metadata-preview.scss

```diff
@@ -202,71 +202,79 @@ ul.sub-field {
   padding: 0;
 }
 
-.collection-tabs {
-  display: flex;
-  flex-wrap: wrap;
-  max-width: 100%;
-  // background: #efefef;
-  // box-shadow: 0 48px 80px -32px rgba(0,0,0,0.3);
-
-  .hidden-input {
-    position: absolute;
-    opacity: 0;
-  }
-
-  .tab-label {
-    background: #e5e5e5;
-    border: 1px solid #c9dce7;
-    color: #7f7f7f;
-    cursor: pointer;
-    font-size: 18px;
-    font-weight: bold;
-    margin-right: 5px;
-    padding: 12px 15px;
-    transition: background 0.1s, color 0.1s;
+#collection-preview-tabs {
+  .collection-tabs-container {
+    border-bottom: 1px solid #c9dce7;
+    box-sizing: border-box;
+    display: flex;
+    flex-wrap: nowrap;
+    // flex-flow: row, nowrap;
+    justify-content: flex-start;
+    // margin-bottom: -1px;
     width: 100%;
-  }
+    // max-width: 100%;
 
-  .tab-label:hover {
-    background: #d8d8d8;
-  }
+    .tab-label {
+      // background: #e5e5e5;
+      background: linear-gradient(to bottom,#fdfdfe 0%,#eff1f4 100%);
+      border: 1px solid #c9dce7;
+      color: #7f7f7f;
+      cursor: pointer;
+      font-size: 16px;
+      font-weight: bold;
+      margin-right: 5px;
+      padding: 12px 15px;
+      transition: background 0.1s, color 0.1s;
+      // width: 100%;
```

Comment:
I think we should clean up the commented css we don't need anymore

Reply:
done